ext/ffi/lib.rs

```diff
@@ -631,6 +666,71 @@ async fn op_ffi_call_ptr_nonblocking(
     .unwrap()
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct FfiGetArgs {
+  rid: ResourceId,
+  name: String,
+  r#type: NativeType,
+}
+
+fn op_ffi_get_static(
+  state: &mut deno_core::OpState,
+  args: FfiGetArgs,
+  _: (),
+) -> Result<Value, AnyError> {
+  let resource = state
+    .resource_table
+    .get::<DynamicLibraryResource>(args.rid)?;
+
+  let data_ptr = resource.get_static(args.name)? as *const u8;
+
+  Ok(match args.r#type {
+    NativeType::Void => {
+      unreachable!();
```

Comment:
Are you throwing in JS land? This is still reachable.

Reply:
No, this is admittedly mostly a `todo!()`. There's probably no reason to define a void type static, so I left this to stew.

Should I just return a void for completeness sake?

Reply:
I'd rather throw JS side because just returning a void isn't really the correct behavior (?)

Reply:
I'm not even sure there is a correct behaviour.